content/news/2021-04-01-mixxx-subscription-model.md

```diff
@@ -0,0 +1,30 @@
+title: "Announcing the Mixxx Subscription Service"
+authors: Jan Holthuis
+tags: subscription, cloud
+date: 2021-04-01 12:00:00
+summary: Starting with Mixxx 2.3, Mixxx will switch to a new, multi-tiered subscription plan.
+
+We are happy to announce a major change in our licensing model:
+Starting with Mixxx 2.3, Mixxx will switch to a new, multi-tiered subscription plan.
+
+![a laptop with the text "Take your DJing to the next level - Introducing the Mixxx subscription service. Prices starting from $6.99/month (introductory offer)"]({static}/images/news/subscription-service.png)
+
+Using our Mixxx **Core** plan for only $6.99/month (introductory offer, $9.99/month after the first 3 months), you can get your party started.
+It offers advanced DJ functions like a top-notch library and tag editing for organizing your music and supports playback of a single deck.
+
+If you want to enjoy advanced DJ features such as using a second deck and crossfader, recording your set, streaming to Shoutcast servers, check out our Mixxx **Creative** plan for only $9.99/month (introductory offer, $14.99/month after the first 3 months).
+
+The Mixxx **Pro** plan ($19.99/month) even goes one step further and lets you spice up your mix by adding EQs, controller support and samplers!
+
+With our additional **FX**, **DVS** and **Spinal Tap** plans ($4.99/month each, can be purchased in addition to any of the Core/Creative/Pro plans) you can take your DJing to the next level and open even more possibilities by adding support for effect chains, enabling support for timecode vinyl and making all knobs and sliders go up to eleven (for that extra push over the cliff).
+
+For real headliners, the **Bottle Service VIP** plan ($499.99/program launch) will guarantee the appearance of at least three Instagram Influencers at each of your parties.
+
+This is an amazing opportunity for us to generate shareholder value and reshape the DJ experience for our customers.
+Because of this is such a huge step forward, we don't want to overwhelm our users and will postpone all bug fixes and features for an indefinite amount of time instead of including them in Mixxx 2.3.
```

Comment:
```suggestion
Because of this is such a huge step forward, we don't want to overwhelm our users and make the transition as easy as possible. Therefore, we will postpone all bug fixes and features for an indefinite amount of time instead of including them in Mixxx 2.3.
```